terraform/aws/components/gp2gp/gpc-consumer-locals_env_variables.tf

```diff
@@ -12,29 +12,13 @@ locals {
       name  = "GPC_CONSUMER_LOGGING_LEVEL"
       value = var.gpc-consumer_log_level
     },
-    {
-      name  = "GPC_CONSUMER_URL"
-      value = "http://${module.gpc-consumer_ecs_service.loadbalancer_dns_name}:${var.gpc-consumer_service_container_port}"
-    },
-    {
-      name  = "GPC_CONSUMER_OVERRIDE_GPC_PROVIDER_URL"
-      value = var.gp2gp_create_gpcapi_mock ? "http://${module.gpcapi_mock_ecs_service[0].loadbalancer_dns_name}:${var.gp2gp_mock_port}" : var.gpc-consumer_override_gpc_provider_url
-    },
-    {
-      name  = "GPC_CONSUMER_GPC_GET_URL"
-      value = var.gp2gp_create_sdsapi_mock ? "http://${module.sdsapi_mock_ecs_service[0].loadbalancer_dns_name}:${var.gp2gp_mock_port}" : var.gpc-consumer_override_gpc_provider_url
-    },
     {
       name  = "GPC_CONSUMER_SDS_URL"
-      value = var.gp2gp_create_sdsapi_mock ? "http://${module.sdsapi_mock_ecs_service[0].loadbalancer_dns_name}:${var.gp2gp_mock_port}" : var.gpc-consumer_sds_url
```

Comment:
Sooooo, it looks like we wouldn't be using the SDS Mock API at all anymore? Did you have any thoughts on whether to remove entirely?

Reply:
The SDS Mock API configuration relies on there being two mocks, the SDS Mock and the GPC Mock.  The "wiremock" mocks both these services.  The environment variables used to configure the consumer adaptor to used for the "two mock" configuration has been removed from the GPC Consumer adaptor.

I think removing the SDS Mock and the GPC Mock from the scripts makes a lot of sense. 

Something else to note is these changes will break other environments defined in these scripts, namely, `build1`. `build 2`, `build3`, `kdev` and `vp`. As these also expect older versions of the adaptor. I'm not sure what the value in maintaining these environments is? Maybe they should also be removed?